src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = days * 40;
+  const mediumTermTime = 3;
+  const mediumTermDiscount = 20;
+  const longTermTime = 7;
+  const longTermDiscount = 50;
+
+  if (days < mediumTermTime) {
+    return basePrice;
+  } else if (days >= mediumTermTime && days < longTermTime) {
+    return basePrice - mediumTermDiscount;
```

Comment:
The discount logic for medium-term rentals is incorrect. According to the task description, the discount should be applied for every day beyond the threshold. Therefore, the discount should be calculated as `mediumTermDiscount * days` instead of a flat `mediumTermDiscount`.